tests/integration/test_charm.py

```diff
@@ -265,6 +306,7 @@ async def test_scale_down(
     assert leader_openfga_integration_data
 
 
+@pytest.mark.skip(reason="the new integration causes failure")
```

Comment:
temporarily disable the charm upgrade test since the one in Charmhub does not have the `smtp` integration. Will put it back on after this PR is merged.

Reply:
I would ask to make the message more specific. It might not be clear what integration causes the failure.